lib/web/ui/usercontext.go

```diff
@@ -1,43 +1,91 @@
 package ui
 
 import (
+	"github.com/gravitational/teleport/lib/defaults"
 	"github.com/gravitational/teleport/lib/services"
-
-	"github.com/gravitational/trace"
+	"github.com/gravitational/teleport/lib/utils"
 )
 
+type access struct {
+	Read   bool `json:"read"`
+	Edit   bool `json:"edit"`
```

Comment:
Minor issue, but why not use `Update` here? Then it's consistent with the verbs on the backend.

Reply:
'edit' brings a better semantic meaning to UI as it's mostly used for content editing. As for 'delete' - this is a reserved word in javascript.